mu4/domain/notation/internal/notationelementsrepository.cpp

```diff
@@ -0,0 +1,22 @@
+#include "notationelementsrepository.h"
+
+#include "libmscore/score.h"
+#include "log.h"
+
+using namespace mu::domain::notation;
+
+NotationElementsRepository::NotationElementsRepository(IGetScore* getScore) :
+    m_getScore(getScore)
+{
+}
+
+Ms::Measure* NotationElementsRepository::measureByIndex(const int measureIndex) const
+{
+    Ms::Score* score = m_getScore->score();
+
+    IF_ASSERT_FAILED(m_getScore && score) {
```

Comment:
The m_getScore check is unnecessary here. A little higher, you have already used it.

Reply:
Agreed